### ISSUE TITLE
Questions - Electron Version and .babelrc

### ISSUE THREAD

**Issue by philcockfield** [shamofu/electron-react-parcel-boilerplate#4]
Hey there @shamofu,

Couple of question popped up as I was working over your (fine) repo:

----

### Electron Version
I noticed you reverted from Electron `v4` => `v3` [[commit](https://github.com/shamofu/electron-react-parcel-boilerplate/commit/f46a234664afd071156f61f14a2e1160c83920a6#commitcomment-32042689
)]

I was wondering why that was.  What it an issue with V4 of electron that would be useful to know about?

----

### .babelrc

I'm wondering why you included a `.babelrc` file.  Parcel seems to pick up these settings automatically, and much of the "electron" target is specified within the command line args in `package.json`.

It also seems to work find without it.  Wondering what you know here that's not obvious 🧐.

---
Thanks @shamofu - again, this repo is proving unbelievably helpful!!!

**Comment by shamofu** [comment]
Hello @philcockfield.

I downgraded Electron in f46a234 because [Azure Pipelines](https://dev.azure.com/shamofu/electron-react-parcel-boilerplate/_build/results?buildId=90) failed. I found the cause was Electron v4.
I'll upgrade Electron when Spectron becomes compatible with Electron v4.

`.babelrc` exists for `babel-jest`. Without `.babelrc`, we can't run Jest.
However, I'm not sure `preset-env` is necessary.
